galata/test/jupyterlab/terminal.test.ts

```diff
@@ -103,3 +103,26 @@ test('Terminal should open in Launcher cwd', async ({ page, tmpPath }) => {
   await page.waitForTimeout(1000);
   expect(await terminal.screenshot()).toMatchSnapshot('launcher-term.png');
 });
+
+test('Terminal web link', async ({ page, tmpPath }) => {
+  await page.waitForSelector(`.jp-Launcher-cwd > h3:has-text("${tmpPath}")`);
+
+  await page.locator('[role="main"] >> p:has-text("Terminal")').click();
+
+  const terminal = page.locator(TERMINAL_SELECTOR);
+  await terminal.waitFor();
+
+  await page.waitForTimeout(1000);
+  await page.keyboard.type('echo https://jupyter.org/');
+  await page.keyboard.press('Enter');
+  await page
+    .locator('[role="region"] canvas')
+    .nth(3)
+    .click({
```

Comment:
Do you mean `hover` instead of `click`?

Reply:
Fixed in [cea839d](https://github.com/jupyterlab/jupyterlab/pull/13645/commits/cea839d0cd4e6f4170cdeb5c82deaef9a7ebb678).